syftlib/src/main/java/org/openmined/syft/networking/clients/MessageProcessor.kt

```diff
@@ -0,0 +1,10 @@
+package org.openmined.syft.networking.clients
+
+import org.openmined.syftproto.types.torch.v1.ScriptModuleOuterClass
+
+class MessageProcessor {
```

Comment:
Will this be going to `jobstatusprocessor` now?

Reply:
It should be invoked from there, yes but it should be its own entity.